doc/architecture/README.md

```diff
@@ -58,6 +58,49 @@ The following applications are supported:
 
 ![differentialDrive](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/BlueAndi/RadonUlzer/main/doc/architecture/uml/LogicalView/DifferentialDrive.plantuml)
 
+Robot angular speed vs wheel linear speed\
+$v [\frac{mm}{s}] = \frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2}$
+
+* $v$: The linear speed of one wheel relative to the other $[\frac{mm}{s}]$.
+* $w_r$: The robot's angular speed $[\frac{rad}{s}]$.
+* $W$: The robot's wheel base $[mm]$.
+* $v_L = -v_R$ by rotation about midpoint of the wheel axis.
+
+Linear speed left\
+$v_L [\frac{mm}{s}] = -\frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2}$
+
+Linear speed right\
+$v_R [\frac{mm}{s}] = \frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2}$
+
+Consider encoder steps per m\
+$v [\frac{steps}{s}] = \frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2} \cdot \frac{ENC [\frac{steps}{m}]}{1000}$
+
+* $v$: The linear speed of one wheel relative to the other $[\frac{steps}{s}]$.
+* $w_r$: The robot's angular speed $[\frac{rad}{s}]$.
+* $W$: The robot's wheel base $[mm]$.
+* $ENC$: The number of encoder steps per m $[\frac{steps}{m}]$.
+
+Linear speed left\
+$v_L [\frac{steps}{s}] = -\frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2} \cdot \frac{ENC [\frac{steps}{m}]}{1000}$
+
+Linear speed right\
+$v_R [\frac{steps}{s}] = \frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2} \cdot \frac{ENC [\frac{steps}{m}]}{1000}$
+
+Consider robot linear speed center\
+
+Linear speed left\
+$v_L [\frac{steps}{s}] = \frac{v_{Linear}}{2} [\frac{steps}{s}] -\frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2} \cdot \frac{ENC [\frac{steps}{m}]}{1000}$
+
+Linear speed right\
+$v_R [\frac{steps}{s}] = \frac{v_{Linear}}{2} [\frac{steps}{s}] + \frac{w_r [\frac{rad}{s}] \cdot W [mm]}{2} \cdot \frac{ENC [\frac{steps}{m}]}{1000}$
+
+Consider angular speed in mrad per s\
+Linear speed left\
```

Comment:
If I'm not mistaken, you are just repeating the equation from line 92 (except for calculation with mrad/s instead of rad/s.) --> why?

Reply:
Because the calculation in the SW is done in mrad/s, and these equations show step by step how the final equation is reached. This is a simple step (divide by 1000) but should remain here.

Reply:
okay.